src/new-client/src/Web/TearOutSection/state.ts

```diff
@@ -0,0 +1,14 @@
+import { bind } from "@react-rxjs/core"
+import { createSignal } from "@react-rxjs/utils"
+import { map, filter } from "rxjs/operators"
+
+type TearOutSectionEntry = [boolean, string]
```

Comment:
As above, this string should use the Section type

Reply:
Done